src/routes/index.tsx

```diff
@@ -7,6 +7,7 @@ import SplashScreen from "~/pages/public/SplashScreen";
 import NavigationWrapper from "~/routes/NavigationWrapper";
 
 export default function AppRoutes() {
+  // TODO validate token on first start here?
```

Comment:
Ist ja im Grunde der Root unserer App, richtig? Wäre daher sinnvoll. Die Logik könnte man ja einfach in einer Hook abbilden. Dann können wir sie im Nachgang auch einfacher verschieben bzw. woanders einsetzen, falls nötig.